src/deployment-guides/tko-on-vsphere.md

```diff
@@ -1,4 +1,5 @@
 # Deploy Tanzu for Kubernetes Operations on vSphere
+<!-- markdownlint-disable MD013 MD033 MD045 MD005 -->
```

Comment:
I was using [markdownlint](https://github.com/markdownlint/markdownlint) locally and wanted to silence the lint cops in my editor. I incorporated it into my workflow after I saw that [tanzu-framework](https://github.com/vmware-tanzu/tanzu-framework) makes use of it for their usage docs. It would be a good idea to add it here if there's consensus on it being a good idea!